kolibri/core/assets/src/views/core-base.vue

```diff
@@ -121,15 +121,19 @@
 
   @require '~kolibri.styles.definitions'
 
-  .loading-spinner-fixed
-    position: fixed
+  align-to-parent() {
+    position: absolute
+    top: 0
+    left: 0
+  }
+
+  .loading-spinner-aligned
+    align-to-parent()
 
   .app-bar
+    align-to-parent()
```

Comment:
FYI up until now we haven't really leveraged Stylus functionality too much and used it primarily for its concise syntax, variable definitions, and occasional nesting.

I don't think that's necessarily a reason to avoid this, but I was curious whether the mixin syntax has any advantages over simply making a new class? If not, might be worth sticking with classes to avoid lock-in?


e.g. 

```
<loading-spinner v-if="loading" class="loading-spinner align-to-parent"/>
```

```css
.loading-spinner
  /* ... */

.align-to-parent
  position: absolute
  top: 0
  left: 0
```